doc/source/timeseries.rst

```diff
@@ -185,6 +185,17 @@ options like ``dayfirst`` or ``format``, so use ``to_datetime`` if these are req
 
     pd.Timestamp('2010/11/12')
 
+You can also use the `DatetimeIndex` constructor directly:
+
```

Comment:
Can you put the first `DatetimeIndex` example here (the one without `freq='infer'`)?